### ISSUE TITLE
Parameterize Load Balancing strategy

### ISSUE THREAD

**Issue by achontzo** [F5Networks/k8s-bigip-ctlr#1865]
#### Title
Parameterize Load Balancing Strategy
#### Description

Load balancing strategy is hard-coded to first-match value giving no option but to build custom images for cases that another LB strategy is needed.

#### Actual Problem
As first-match is the only available option, creating ingress resources that uses same host may lead to a situation were traffic is routed to wrong pool members.

A scenario that best-match is needed follows:
Create first deployment1 with an ingress resource that uses:
```
- host: www.myhost.com
- path: /
```
Now create deployment2 with an ingress resource that uses the same host but different path:
```
- host: www.myhost.com
- path: /path

```
F5 CIS will evaluate and reconfigure LTM by the same order that the ingress objectes were applied. The rules inside LTM published policy are in ascending order, meaning that LTM will first evaluate policy created by (1). As Strategy is "execute first matching rule" it will match / but also /path requests with an outcome that all traffic will be forwarded to deployment1.

 
#### Solution Proposed
Parameterize Load Balancing Strategy.

**Comment by trinaths** [comment]
Created JIRA CONTCNTR-2773 for internal tracking

**Comment by trinaths** [comment]
This issue is addressed in 2.4.1. PR https://github.com/F5Networks/k8s-bigip-ctlr/pull/1800 fixed this issue. 

Please upgrade and verify with 2.4.1 and share your findings.

**Comment by trinaths** [comment]
@achontzo This issue is addressed in 2.4.1. PR #1800 fixed this issue.

Please upgrade and verify with 2.4.1 and share your feedback

**Comment by trinaths** [comment]
closing this issue.
